src/status_im/contexts/shell/share/profile/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im.contexts.shell.share.profile.view
```

Comment:
created a new file share profile view  toimprove code organization and readability.